packages/client/src/app/game/views/game-container/game-toolbar/game-toolbar.component.ts

```diff
@@ -6,10 +6,10 @@ import { MatDialog } from '@angular/material';
   selector: 'game-toolbar',
   template: `
     <div class="right-btn-panel">
-      <button mat-icon-button class="help-btn" (click)="toggleMute($event)">
+      <button mat-icon-button class="help-btn" (click)="toggleMute()">
```

Comment:
ERROR in ng:///home/ubuntu/CesiumFPS/packages/client/src/app/game/views/game-container/game-toolbar/game-toolbar.component.ts.G  eToolbarComponent.html (4,48): Supplied parameters do not match any signature of call target.

ERROR in ng:///home/ubuntu/CesiumFPS/packages/client/src/app/game/views/game-container/game-toolbar/game-toolbar.component.ts.G  eToolbarComponent.html (7,48): Supplied parameters do not match any signature of call target.

ERROR in /home/ubuntu/CesiumFPS/packages/client/src/$$_gendir/app/game/views/game-container/game-toolbar/game-toolbar.component  gfactory.ts (94,36): Property 'mute' is private and only accessible within class 'GameToolbarComponent'.